main/main.c

```diff
@@ -30,13 +30,14 @@
 #include <unistd.h>
 
 static void usage(const char *prog) {
-	printf("Usage: %s [-h] [-v] [-t] [-c FILE] [-s PATH]\n", prog);
```

Comment:
I prefer alphabetical order. It is probably stupid, but it leaves no place for interpretation to "Where should I put my new option?".

That's a global comment on how the options parsing is done in this file.

Reply:
Ack, I'll change that in a separate commit. I will also update the man page stub.

Reply:
The -h --help flag should always be first though. Don't you think?

Reply:
No exception, plus -h is quasi standard and a user should not be looking for it (hopefully).

Reply:
Allez, d'accord.